svc/GridExportService.js

```diff
@@ -112,6 +112,8 @@ export class GridExportService {
     // Implementation
     //-----------------------
     getExportableColumns(gridModel, columns) {
+        if (isFunction(columns)) return columns(gridModel);
```

Comment:
The function form is documented as returning column Ids, but that wouldn't work here, since `getExportableColumns` return column instances. I think the function form *should* return ids, but we've have to do a bit of work here to get the instances from the GridModel.

Reply:
great catch -- fixed this, and tested properly.

Note that the testing also helped me find and fix an unrelated problem in the new sorting code that you should also review as well